docker/Dockerfile

```diff
@@ -0,0 +1,145 @@
+## pick a specific version, to ensure predictability
+FROM jupyter/pyspark-notebook@sha256:64420e4c348ab48fb806f42332109cbc205ae74cda67240c3e0974c5f7e6e969
+## or latest
+#FROM jupyter/pyspark-notebook@latest
+
+MAINTAINER Intel BigDL Project <bigdl@intel.com>
+
+## --- CONFIG
+ARG MAVEN_VERSION=3.5.2
+ARG SCALA_VERSION=2.11.8
+ARG SPARK_VERSION=2.2.0
+ARG SBT_VERSION=1.0.2
+ARG INSTALL_DIR=/usr/local
```

Comment:
There is WORKDIR  instruction, with it you don't need to do "cd ${INSTALL_DIR}"

Reply:
INSTALL_DIR is used for setting many PATH envs.